src/store/reducers/devicesReducer.js

```diff
@@ -81,7 +81,12 @@ const deleteDevicesReducer = (state, { payload }) => {
 };
 
 const insertDevicesReducer = (state, { payload }) => {
-  const { entities: { devices }, result } = normalize(payload, deviceListSchema);
+  const inserted = payload.reduce((acc, payloadItem) => {
```

Comment:
The naming `showGlimmer` seems a bit too bound to the particular implementation of the real underlying idea - namely,  that the data is "new". I think a better name could be used in the data to indicate that the row is fresh or new.